### PR TITLE
SwiftStorage: log failed connections / authentications

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/PLATFORM-890

Log something more meaningful than:

```
PHP Fatal Error: Exception from line 211 of /usr/wikia/slot1/3445/src/extensions/SwiftCloudFiles/php-cloudfiles-1.7.10/cloudfiles.php: Unexpected response (): http://10.x.x.x:8060/auth (curl error: 7) couldn't connect to host http://10.x.x.x:8060/auth (http://math.wikia.com/wiki/Fraction?action=submit) in /usr/wikia/slot1/3445/src/includes/wikia/nirvana/WikiaException.php on line 33
```

@michalroszka

### CODE REVIEW

File: includes/wikia/SwiftStorage.class.php [hunk 1]
```diff
@@ -90,11 +90,22 @@ private function __construct( $containerName, $pathPrefix = '', $dataCenter = nu
...
-		$this->container = $this->getContainerObject( $containerName );
 		$this->containerName = $containerName;
 		$this->pathPrefix = rtrim( $pathPrefix, '/' );
+
+		try {
+			$this->connect( $this->swiftConfig );
+		}
+		catch( \Exception $ex ) {
+			$this->error( 'SwiftStorage: connect failed', [
+				'exception'  => $ex,
```
Comment (wladekb) [inline includes/wikia/SwiftStorage.class.php]: is it enough? we don't need to log container or prefix too?

Comment (macbre) [inline includes/wikia/SwiftStorage.class.php]: Scroll down to `getLoggerContext` method :)

Comment (wladekb) [inline includes/wikia/SwiftStorage.class.php]: Oh, it's a nice feature I wasn't aware of.

**Comment by wladekb** [comment]
lgtm

**Pull request merged**